### PR TITLE
feat - study group 

### PR DESCRIPTION
**Pull request by distalx**
Study group will allow people to organize their hangouts better!

### CODE REVIEW

**Comment by lpatmo** [comment]
@sergeant-q Any chance you could take a look? Realize there's a lot...

**Comment by sergeant-q** [comment]
This is a big commit! I will do my best to review the changes. This brings up a good point about using a tool to create and track user stories. I'll discuss on Slack further.

**Comment by lpatmo** [comment]
## Study group specs
Tabs: 24/7 hangout, All Hangouts, Members, Learnings, Links, and Activity

## Problem solved
Many people, especially new members, don’t want to create a hangout right away. Study groups allow people to organize better (not just in Slack, where there are *way* too many channels) so that they can create or sign up to join a study group asynchronously. Then, it’ll be easier to organize hangouts within those study groups.

### User flow
- Create a new study group from the navbar, with fields "name" and "tagline"

## On the study group itself
### Member view:
1. Click to join the study group.
2. After joining, can type in "What did you learn?" box to share what you learned. These learnings will be tracked in the "Learnings" tab.
3. Ability to leave the study group by clicking on a button.
4. Seeing a list of organizers (people who have permissions to schedule a hangout for the study group or add links to the Links tab) 
5. Ability to join a 24/7 hangout by clicking to load the hangout.
6. Ability to upvote learnings collected in the study group.
7. Click to leave the study group.

### Owner view:
Ability to do everything a member can, plus:
1. Schedule a hangout for the group.
2. Add links to the group.
3. Edit the description and goals of the group.
4. Make other users organizers (`moderator` or `manager`) in the Members tab by clicking on the user icon.
5. No ability to leave the study group.

### Moderator view:
Ability to do everything a member can, plus:
1. Schedule a hangout for the group.
2. Add links to the group.

### Manager view:
Ability to do everything a member can, plus:
1. Schedule a hangout for the group.
2. Add links to the group.
3. Edit the description and goals of the group.
4. Ability to make other people managers so that they can adjust permission settings for other users as well.

File: client/css/_study_groups.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  }
+  p.overflowtext {
+    margin: 0px;
+  }
+  #info .panel-body {
+  	padding: 10px 0px;
+  }
+}
+
+.newStudyGroup {
```
Comment (sergeant-q) [inline client/css/_study_groups.scss]: I am noticing inconsistent naming conventions for our classes. We need to stick to one.

File: client/templates/hangout/hangout-frame.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ Template.hangoutFrame.onCreated(function() {
     instance.api.executeCommand('displayName', data.username);
     instance.api.executeCommand('toggleChat');
     instance.api.executeCommand('avatarUrl', data.avatar);
+    let jitsiParticipants = instance.api.getNumberOfParticipants();
+    console.log(jitsiParticipants);
```
[the remark this answers is not in the document]
Comment (lpatmo) [reply client/templates/hangout/hangout-frame.js]: Thanks @sergeant-q. I think I removed it in a recent commit here. This is the most up-to-date diff: https://github.com/codebuddiesdotorg/codebuddies/pull/557/files#diff-d55d625faf23e09fb8d8590973e90ae4

**Pull request merged**